# shape_type: keyword of one of the figure types that the laser projector system is able to project:
# - polyline
# - curves: circle, oval, arc (todo)
# - text (todo)
string shape_type

# Group name that the projection element belongs
# Several projection elements can be handled as a projection group
string group_name

# Name of the projection element
string shape_id

---

bool success
string message